code/game/machinery/computer/medical.dm

```diff
@@ -24,8 +24,18 @@
 /obj/machinery/computer/med_data/attack_paw(user as mob)
 	return src.attack_hand(user)
 
-/obj/machinery/computer/med_data/attack_hand(mob/user as mob)
-	if(..())
+/obj/machinery/computer/med_data/attack_hand(var/mob/user)
+	. = ..()
+
+	if(stat & (BROKEN|NOPOWER))
+		if (scan)
```

Comment:
for most machines the ID is stuck when unpowered unless you kick it til it pops out